lib/elixir/lib/calendar/datetime.ex

```diff
@@ -504,6 +506,39 @@ defmodule DateTime do
     shift_zone(utc_now(), time_zone, time_zone_database)
   end
 
+  @doc """
+  Returns the current datetime in the provided time zone or raises on errors
+
+  See `now/2` for more information.
+
+  ## Examples
+
+      iex> datetime = DateTime.now!("Etc/UTC")
+      iex> datetime.time_zone
+      "Etc/UTC"
+
+      iex> DateTime.now!("Europe/Copenhagen")
+      ** (ArgumentError) cannot get current datetime in "Europe/Copenhagen" time zone, reason: :utc_only_time_zone_database
+
+      iex> DateTime.now!("bad timezone", FakeTimeZoneDatabase)
+      ** (ArgumentError) cannot get current datetime in "bad timezone" time zone, reason: :time_zone_not_found
+
+  """
+  @doc since: "1.10.0"
+  @spec now!(Calendar.time_zone(), Calendar.time_zone_database()) :: t
+  def now!(time_zone, time_zone_database \\ Calendar.get_time_zone_database()) do
+    case now(time_zone, time_zone_database) do
+      {:ok, datetime} ->
+        datetime
+
+      {:error, reason} ->
+        raise ArgumentError,
+              "cannot get current datetime in #{inspect(time_zone)} time zone, reason: #{
+                inspect(reason)
+              }"
```

Comment:
```
"cannot get current datetime in #{inspect(time_zone)} time zone, reason: " <>
  inspect(reason)
```

Reply:
yeah, already changed but forgot to push!